modules/imgcodecs/src/grfmt_exr.cpp

```diff
@@ -649,12 +650,14 @@ bool  ExrEncoder::write( const Mat& img, const std::vector<int>& params )
             case IMWRITE_EXR_COMPRESSION_B44A:
                 header.compression() = B44A_COMPRESSION;
                 break;
+#if (OPENEXR_VERSION_MAJOR >= 2) && (OPENEXR_VERSION_MINOR >= 2)
```

Comment:
I think this check would not work with `OPENEXR_VERSION_MAJOR=3` and `OPENEXR_VERSION_MINOR=0`.

Reply:
Fixed.